### PR TITLE
Feature 138 casestudytemplates

### PR DESCRIPTION
**Pull request by ryleyw**
Addition of the individual case study pages

### CODE REVIEW

File: react-frontend/src/components/Home/infocard.js [hunk 1]
```diff
@@ -20,12 +20,12 @@ function InfoCard({
     isRouteCard = true;
   }
 
-  cardHeight = height == null ? '300px' : height;
+  cardHeight = height == null ? '600px' : height;
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/Home/infocard.js:23]: null is falsy so the simpler way of coding this is 
cardHeight = height ? height :  '600px' ;

File: react-frontend/src/components/CaseStudies/cardlist.js [hunk 2]
```diff
@@ -14,13 +15,8 @@ const descriptions = [
   'In 2016, MSP received 130,000 enrolment and 147,680 Premium Assistance applications. Thirty percent of application forms were returned due to errors. To improve the process, the...',
 ];
 
-const imgLinks = [
-  'https://dalehartery.files.wordpress.com/2015/05/light-grey-background-pattern.jpg',
-  'https://dalehartery.files.wordpress.com/2015/05/light-grey-background-pattern.jpg',
-  'https://dalehartery.files.wordpress.com/2015/05/light-grey-background-pattern.jpg',
-];
-
 function CardList() {
```
Comment (patricksimonian) [inline react-frontend/src/components/CaseStudies/cardlist.js:18]: like to see this using the arrow functions. Lets be consistent with code style and modern js

File: react-frontend/src/components/CaseStudies/cardlist.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Col, Row, Grid } from 'react-flexbox-grid';
 import InfoCard from '../Home/infocard';
 import HorizontalInfoCard from './horizontalinfocard';
+import { useRouteMatch } from 'react-router-dom';
```
[the remark this answers is not in the document]
Comment (ryleyw) [reply react-frontend/src/components/CaseStudies/cardlist.js:5]: Not sure where to put this

[the remark this answers is not in the document]
Comment (patricksimonian) [reply react-frontend/src/components/CaseStudies/cardlist.js:5]: I think he means just about your relative imports. so between line 2-3

File: react-frontend/src/components/CaseStudies/casestudies.js [hunk 4]
```diff
@@ -1,18 +1,37 @@
...
+      <Switch>
+        <Route exact path={path}>
+          <div className="caseStudyContainer">
+            <CaseStudiesBanner />
+            <div className="caseStudyBody">
+              <CardList />
+            </div>
+          </div>
+        </Route>
+        <Route path={path + '/mining-study'}>
```
Comment (patricksimonian) [inline react-frontend/src/components/CaseStudies/casestudies.js:23]: https://medium.com/better-programming/how-to-pass-multiple-route-parameters-in-a-react-url-path-4b919de0abbe#:~:text=dynamic%20content%20required.-,Route%20parameters,data%20based%20on%20the%20parameter.

Comment (patricksimonian) [reply react-frontend/src/components/CaseStudies/casestudies.js:23]: just a reference to a more DRY approach!

File: react-frontend/src/css/casetemplates.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+    .pageContainer {
+        padding-top: 65px;
+    }
+    .pageBody {
+        width: 330px;
+    }
+
+    .cardAdjustment {
+        margin-left: -8px;
```
Comment (patricksimonian) [inline react-frontend/src/css/casetemplates.css:28]: this hurts me but im okay with it now. Negative margins are not used too often for positioning

File: react-frontend/src/css/casetemplates.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    padding-bottom: 70px;
+}   
+
+.pageBody {
+    width: 85%;
+    margin: auto;
+}
+
+.pageSubtitle {
+    font-size: 22pt;
```
Comment (patricksimonian) [inline react-frontend/src/css/casetemplates.css:13]: pxs not pts

**Pull request merged**